style2.css

```diff
@@ -0,0 +1,716 @@
+    color: #6f7073;
+    padding-left: 13px;
+  }
+
+  .foot .la {
+    font-family: 'Lato', sans-serif;
+    font-size: 18px;
+    color: #6f7073;
+    padding-left: 13px;
+  }
+
+  .para1 {
+    background-color: white;
+
+    --bs-gutter-y: 0;
+
+    padding: 20px;
+    margin: 15px 23px 14px 12px;
+    border: 2px solid #8080804d;
+  }
+
+  .para1 p {
+    margin: 1.0625rem 0.75em 0;
+    width: calc(100% - 2.5em);
+    padding: 1em 0.5em;
+    line-height: 150%;
+    font-weight: normal;
+    font-size: 0.8125rem;
+    font-family: var(--ff-lato);
+    height: auto;
+    text-align: justify;
+    border: 0.25rem solid var(--clr-neutral-white);
+  }
+
+  .para2 {
+    padding: 2.5rem 2.5rem 0.625rem 2.5rem;
+  }
+
+  .second {
+    padding: 2.5rem 2.5rem 0.625rem 2.5rem;
+    font: 700 0.875rem var(--ff-lato);
+  }
+
+  .second a {
+    color: black;
+  }
+
+  .container-back {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 2em;
+    border: 1px solid rgb(241, 238, 241);
+  }
+
+  .container {
+    flex-direction: column;
+    gap: 1em;
+    background-color: white;
+    padding: 3.1875rem 1.5625rem 0 1.5625rem;
+  }
+
+  .strong {
+    text-align: center;
+    font-size: 1.5rem;
+  }
+
+  .light {
+    font-family: 'Lato', sans-serif;
+    font-size: 1rem;
+    color: #6f7073;
+    text-align: center;
+  }
+
+  hr {
+    width: 10%;
+    height: 0.0625rem;
+    background-color: orangered;
+    margin: 0.5rem auto;
+  }
+
+  .past-summit {
+    margin-top: 1em;
+    flex-direction: column;
+    gap: 1em;
+  }
+
+  .image {
+    width: 31em;
+    background-color: white;
+
+    --bs-gutter-y: 0;
+
+    padding: 20px;
+    margin: 15px 23px 14px 12px;
+    border: 2px solid #8080804d;
+  }
+
+  .image img {
+    padding-left: 39px;
+  }
+
+  .str {
+    font-size: 1.5rem;
+    align-items: center;
+  }
+
+  .lig {
+    font-family: 'Lato', sans-serif;
+    font-size: 18px;
+    color: #6f7073;
+    align-items: center;
+  }
+
+  .cut {
+    font-family: 'Lato', sans-serif;
+    font-size: 18px;
+    color: #6f7073;
+  }
+
+  .history {
+    display: flex;
+    flex-direction: column;
+    gap: 2em;
+  }
+
+  .year {
+    padding-top: 61px;
+    display: flex;
+    align-self: center;
+    color: white;
+    font-size: 1.5em;
+  }
+
+  .cc {
+    padding-left: 0.5em;
+    color: white;
+    font-size: 24px;
+    padding-right: 0.5em;
+  }
+
+  .first-histo {
+    flex-direction: column;
+    background-image: url(img/hawa.jpg);
+    background-repeat: no-repeat;
+    background-size: 100% auto;
+    height: 20em;
+    padding: 0;
+    margin: 0;
+    filter: opacity(0.5) drop-shadow(0 0 rgb(255, 4, 4));
+  }
+
+  .sec-histo {
+    flex-direction: column;
+    background-image: url(img/fem.jpeg);
+    background-repeat: no-repeat;
+    background-size: 100% auto;
+    height: 20em;
+    padding: 0;
+    margin: 0;
+    filter: opacity(0.5) drop-shadow(0 0 rgb(255, 4, 4));
+  }
```

Comment:
As requested by the previous CR also make the text center vertically to make it look alike to the design.

Moreover, kindly apply the filter overlay on the image not on the text so the text appears in its original color not the color of the overlay.

### YOUR DESIGN:-  

![image](https://user-images.githubusercontent.com/94223367/235297832-d4985a2e-2958-4da8-8947-9f170826a0e4.png)


### ORIGINAL DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/235297837-6b97a239-9e53-405e-bb84-8822a9863220.png)